src/outlineView.ts

```diff
@@ -111,16 +118,25 @@ export class OutlineView {
     }
 
     // add new cursorOn attribue
-    const cursorPoint = newBufferPosition.row
+    const cursorPoint = cursor.getBufferRow()
     this.focusedElms = this.pointToElementsMap.get(cursorPoint)
 
     if (this.focusedElms !== undefined) {
+      this.focusedElms[0].scrollIntoView({
+        block: "center", // scroll until the entry is in the center of outline
+      })
       for (const elm of this.focusedElms) {
         elm.toggleAttribute("cursorOn", true)
-        elm.scrollIntoView({
-          block: "center", // scroll until the entry is in the center of outline
-        })
       }
+      // remove focus once cursor moved
+      const disposable = cursor.onDidChangePosition(() => {
```

Comment:
This looks pretty gnarly. Especially if cursor gets deleted (due to multiple cursors it's entirely probable it might). I would strongly suggest binding to `TextEditor.onDidChangeCursorPosition` once and just aborting early if there is nothing to do. If a handler doesn't do anything, it doesn't have a tangible performance impact.

Reply:
Fixed in https://github.com/atom-community/atom-ide-outline/pull/98/commits/3e0c6d04dfdadd60d4bd1c32d7966ac694ade220